### ISSUE TITLE
Docs for vale-action are misleading with filter_mode

### ISSUE THREAD

**Issue by RobPasMue** [errata-ai/vale-action#66]
By looking at the README.md

```md
### `filter_mode` (default: added)

Set the [filter mode](https://github.com/reviewdog/reviewdog#filter-mode) for
`reviewdog`.

...
with:
  # added, diff_context, file, nofilter
  level: nofilter
...
```

It looks like the YAML code snippet is just copypasted from the above. The key should be `filter_mode` and the values, any of the possible options.

**Comment by jdkato** [comment]
Fixed by #67.

**Comment by RobPasMue** [comment]
Thanks @jdkato!
